totp/__init__.py

```diff
@@ -41,20 +42,27 @@ def get_pass_entry(path):
 
 
 def copy_to_clipboard(text):
-    selection = os.environ.get('PASSWORD_STORE_X_SELECTION', 'clipboard')
     try:
+        if platform.system() == 'Linux':
+            selection = os.environ.get('PASSWORD_STORE_X_SELECTION', 'clipboard')
+            command = ['xclip', '-selection', selection]
+        elif platform.system() == 'Darwin':
+            command = ['pbcopy']
+        else:
+            print('Copy to clipboard is not supported on this platform', file=sys.stderr)
+            return
+
         p = subprocess.Popen(
-            ['xclip', '-selection', selection],
+            command,
             stdin=subprocess.PIPE,
             stdout=subprocess.PIPE,
             stderr=subprocess.PIPE
         )
+        p.stdin.write(text)
+        p.stdin.close()
+        p.wait()
     except FileNotFoundError:
-        print('xclip not found. Not copying code', file=sys.stderr)
-
-    p.stdin.write(text)
-    p.stdin.close()
-    p.wait()
+        print('{} not found. Not copying code'.format(command[0]), file=sys.stderr)
```

Comment:
I didn't understand this question (maybe I was too asleep when I replied yesterday 😅 ).